components/TripList.js

```diff
@@ -1,8 +1,13 @@
 import { Container, Card, Header, Footer } from "../styles/styles";
 import Link from "next/link";
 import styled from "styled-components";
+import DeleteTripButton from "./DeleteTripButton";
+
+export default function TripList({ campingTrips, onDeleteTrip }) {
+  const handleDeleteTrip = (tripId) => {
+    onDeleteTrip(tripId);
+  };
```

Comment:
Good news, you can pass the `onDelteTrip` function directly to your component without wrapping it in an additional handler function. 🥳

```suggestion

```